metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/utils/InfluxDbWriteObjectSerializer.java

```diff
@@ -84,8 +84,8 @@ private void formattedTime(Long time, TimeUnit precision, StringBuilder stringBu
     }
 
     private String escapeKey(String key) {
-        String toBeEscaped = SPACE.matcher(key).replaceAll("\\ ");
-        toBeEscaped = COMMA.matcher(toBeEscaped).replaceAll("\\,");
+        String toBeEscaped = SPACE.matcher(key).replaceAll("\\\\ ");
+        toBeEscaped = COMMA.matcher(toBeEscaped).replaceAll("\\\\,");
         return EQUAL.matcher(toBeEscaped).replaceAll("\\=");
```

Comment:
It makes sense not to include `=` in the keys, then we should remove `=` from the keys